vignettes/articles/Hector-Wider-World.Rmd

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Hector in the Wider World"
 author: "Leeya Pressburger"
-date: "7/30/2021"
+date: "11/7/2022"
```

Comment:
Change to auto-generated date?

Reply:
conflicted because the auto-update would update every time the HTML is generated not necessarily when the actual last update to the document content. I would probably opt to drop the date entirely over the auto-generate date.... thoughts?

Reply:
Oh, good point. I retract the suggestion — it's more important to document when _the content_ was updated, as you say. OK, then one other suggestion: change to YYYY-MM-DD?

Reply:
sounds good to me!